src/common.c

```diff
@@ -1283,7 +1276,7 @@ ssize_t ofi_bsock_recv(struct ofi_bsock *bsock, void *buf, size_t len)
 		return bytes;
 	}
 
-	ret = ofi_recv_socket(bsock->sock, buf, len, MSG_NOSIGNAL);
+	ret = ofi_recvv_socket(bsock->sock, buf, len, MSG_NOSIGNAL);
```

Comment:
This has mismatched parameters -- buf and len need to be converted to an iov.

Reply:
Oops, thanks for catching that! This line was not supposed to be changed.